server/meeting_summarization.go

```diff
@@ -161,9 +161,17 @@ func (p *Plugin) newCallTranscriptionSummaryThread(requestingUser *model.User, t
 			return fmt.Errorf("unable to read calls file: %w", err)
 		}
 
-		transcription, err := subtitles.NewSubtitlesFromVTT(transcriptionFileReader)
-		if err != nil {
-			return fmt.Errorf("unable to parse transcription file: %w", err)
+		var transcription *subtitles.Subtitles
+		if transcriptionFilePost.Type == "custom_zoom_chat" {
```

Comment:
This is kinda an awkward way to do this. It's not really a meeting we are summarizing here, just the chat from it. 
I wonder if we could just incorporate the chat into the main summary somehow?

Reply:
We could try that, but my concern is sometimes the chat deviates from the regular conversation and it is out-of-sync with the regular conversation. I think having a summary of both independently can work well. Also, the transcription and the chat are delivered in different webhooks, so it makes the things way more complicated. I would suggest to go with this approach and we can address that later and have another thing that is combining both. But I think for now this is good enough.